pdns/recursordist/RECURSOR-MIB.txt

```diff
@@ -1032,6 +1035,21 @@ udp6InCsumErrors OBJECT-TYPE
         "Number of UDP6 in checksum errors (Linux only)"
     ::= { stats 123 }
 
+unauthorizedNotify OBJECT-TYPE
+    SYNTAX Counter64
+    MAX-ACCESS read-only
+    STATUS current
+    DESCRIPTION
+        "Number of unauthorized NOTIFY operations"
```

Comment:
please mention if this about the domain or IP

Reply:
Done, pending discussion in comments about the sibling metric.